modin/pandas/test/utils.py

```diff
@@ -669,6 +677,22 @@ def df_equals(df1, df2):
             np.testing.assert_almost_equal(df1, df2)
 
 
+def assert_index_equal(left, right, **kwargs):
+    kwargs["exact"] = False
```

Comment:
Why do we have to do this?

Reply:
If exact is not False, pandas implementation of assert_index_equal() compares index classes by identity and this check fails for the index wrapper.